vultr/resource_reverse_name.go

```diff
@@ -0,0 +1,182 @@
+package vultr
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceReverseName() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceReverseNameCreate,
+		Read:   resourceReverseNameRead,
+		Update: resourceReverseNameUpdate,
+		Delete: resourceReverseNameDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"ip": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
It looks like `k` and `d` are unused in this func. Can you replace the parameter names with `_` so it’s clearer?